portiaui/app/components/extracted-items-status.js

```diff
@@ -0,0 +1,42 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+    tagName: '',
+    extractedItems: Ember.inject.service(),
+
+    type: Ember.computed.readOnly('extractedItems.type'),
+    changes: Ember.computed('extractedItems.changes', function() {
+        let changes = this.get('extractedItems.changes');
+        return changes ? changes : [];
+    }),
+
+    hasChanges: Ember.computed('changes', function() {
+        return this.get('changes').length > 0;
+    }),
+
+    hasWarning: Ember.computed('type', function() {
+        if ((this.get('type') === 'js' && this.get('changes').length < 1) ||
+                this.get('hasChanges')) {
+            return true;
+        }
+        return false;
```

Comment:
`return (this.get('type') === 'js' && this.get('changes').length < 1) || this.get('hasChanges');`